src/fsharp/fsharp.core.netcore.nuget/layoutfscorenuget.fsx

```diff
@@ -57,7 +57,7 @@ try
         if not (Directory.Exists(output)) then Directory.CreateDirectory(output) |>ignore
         ()
 
-    let fsharpCompilerFiles =
+    let fsharpCoreFiles =
         seq {
             yield Path.Combine(bindir, "FSharp.Core.dll")
             yield Path.Combine(bindir, "fsharp.core.sigdata")
```

Comment:
I will make it so.


From: Don Syme [mailto:notifications@github.com]
Sent: Tuesday, March 29, 2016 5:08 PM
To: Microsoft/visualfsharp <visualfsharp@noreply.github.com>
Cc: Kevin Ransom <Kevin.Ransom@microsoft.com>
Subject: Re: [Microsoft/visualfsharp] Reduce coreclr dependencies (#1022)


In src/fsharp/fsharp.core.netcore.nuget/layoutfscorenuget.fsx<https://github.com/Microsoft/visualfsharp/pull/1022#discussion_r57821309>:

> @@ -57,7 +57,7 @@ try

>          if not (Directory.Exists(output)) then Directory.CreateDirectory(output) |>ignore

>          ()

>

> -    let fsharpCompilerFiles =

> +    let fsharpCoreFiles =

>          seq {

>              yield Path.Combine(bindir, "FSharp.Core.dll")

>              yield Path.Combine(bindir, "fsharp.core.sigdata")

Why is fsharp.core.sigdata lower case here? On Linux we'd expect this file to be called FSharp.Core.sigdata?

—
You are receiving this because you authored the thread.
Reply to this email directly or view it on GitHub<https://github.com/Microsoft/visualfsharp/pull/1022/files/55153bcfff8ca36390e812225cc524aedb467d33#r57821309>